.gitlab-ci.yml

```diff
@@ -205,11 +205,14 @@ test-lin-dev-clang-cran:
 
 # stated dependency on R
 test-lin-ancient-cran:
-  image: registry.gitlab.com/jangorecki/dockerfiles/r-3.3.0
   <<: *test-lin
+  image: registry.gitlab.com/jangorecki/dockerfiles/r-3.3.0
+  variables:
+    _R_CHECK_FORCE_SUGGESTS_: "FALSE"
   script:
     - *install-deps
-    - R CMD check --no-manual $(ls -1t data.table_*.tar.gz | head -n 1)
+    - Rscript -e 'install.packages("https://cran.r-project.org/src/contrib/Archive/xts/xts_0.12.2.tar.gz", repos = NULL, type = "source")'
```

Comment:
What happens if we don't install xts? Shouldn't `check` pass?

Reply:
Yes, with the added `_R_CHECK_FORCE_SUGGESTS_: "FALSE"`, `check` should pass!